packages/sn-client-core/src/Repository/Repository.ts

```diff
@@ -301,6 +303,25 @@ export class Repository implements Disposable {
     })
   }
 
+  /**
+   * Shares a content or content collection with a specified
+   * @param options Options for the Copy request
+   */
+  public async share(options: SharingOptions): Promise<ODataSharingResponse> {
+    return await this.executeAction<{}, ODataSharingResponse>({
+      idOrPath: options.content.Id,
+      method: 'POST',
+      name: 'Share',
+      body: {
+        token: options.identity.toString(),
+        content: options.content,
+        level: options.sharingLevel,
+        mode: options.sharingMode,
+        sendNotification: options.sendNotification || true,
```

Comment:
See my comment below.
```suggestion
        sendNotification: options.sendNotification ?? true,
```